query.go

```diff
@@ -137,7 +137,6 @@ func (this *Query) SetMaxRetry(count int) *Query {
 }
 
 func _QueryShouldRetry(logID string, status Status) _ExecutionState {
-	logCtx.Trace().Str("requestId", logID).Str("status", status.String()).Msg("query precheck status received")
```

Comment:
To remove `_QueryShouldRetry` - `logID` there will be changes in all transaction files ~20 files.